src/movieData.js

```diff
@@ -0,0 +1,406 @@
+const movieData = {
```

Comment:
It looks like you've added this file, but we're supposed to be refactoring to use the fetch calls, so you should delete this if your fetches are working 😄